### ISSUE TITLE
Doctype?

### ISSUE THREAD

**Issue by jedisct1** [Stebalien/horrorshow-rs#11]
Hi there,

And thanks for Horrorshow!

Just a dumb question: how to insert the document type (`<!doctype html>`)?

**Comment by Stebalien** [comment]
I usually just use `: raw!("<!DOCTYPE html>");`:

```rust
html! {
    : raw!("<!DOCTYPE html>");
    html(lang="en") {
        head { /* ... */ }
        body { /* ... */ }
    }
}
```

However, that's not really a nice solution. What do you think about some constants/helpers for common template items. For example,

```
html! {
    : doctype::HTML5;
    html(lang="en") {
        head {
            : meta::Utf8;
            : meta::Author("My Name");
        }
        body { /* ... */ }
    }
}
```

**Comment by stalkerg** [comment]
For me is nice solution.

**Comment by jedisct1** [comment]
I'd love to see the second solution as well!

**Comment by Stebalien** [comment]
This should be fixed in `0.6.0` (minor version bump because I changed `Raw` slightly).

**Comment by Stebalien** [comment]
Also, thanks for the feedback!
